tests/TestHierarchy.hs

```diff
@@ -0,0 +1,26 @@
+module TestHierarchy where
+
+import System.Process (readCreateProcess, shell)
+
+import Test.Hspec (Spec, describe, hspec, it, shouldBe)
+
+import TestUtils (pushd)
+
+main :: IO ()
+main = pushd "tests/support/hierarchy" (hspec spec)
+
+spec :: Spec
+spec = do
+  describe "hierarchy" $ do
+    it "generates usable graphviz graphs" $ do
+      let expected = unlines
+            [ "digraph Main {"
+            , "  A;"
+            , "  A -> B;"
+            , "  C;"
+            , "}"
+            ]
+
+      actual <- readCreateProcess (shell "purs hierarchy src/Main.purs") ""
```

Comment:
I think I'd rather not involve a shell in this test, actually. How about refactoring the `App.Command.Hierarchy` module to extract a function `[Module] -> Text`, putting that function into the library portion of the purescript package, and just exercising that function in these tests?